.eslintrc.json

```diff
@@ -1,3 +0,0 @@
-{
-  "extends": "next/core-web-vitals"
```

Comment:
do we want to enforce eslint configs in our repo? Could this have any adverse or unexpected affects

Reply:
I love eslint and find it super helpful. I don't know of any risks because we can always disable eslint line by line as necessary if we see it doing something we don't want.  

E.g. one case is that we care about the order of columns in CSV output, so we don't want it to alphabetize the properties of an object that is getting converted to CSV output. So we disable that specific eslint rule there in that function.

Reply:
I agree they are useful. However, enforcing your personal development configurations in a repo doesn't seem necessary